stock_manager/tests/test_utils.py

```diff
@@ -0,0 +1,107 @@
+import logging
+from unittest.mock import MagicMock
+
+import pytest
+import qrcode.image.base
+
+import stock_manager
+from stock_manager import DatabaseUpdateType, DBUtils, ExportUtils, Item, Logger
+
+
+class TestDatabase:
+    @pytest.fixture
+    def database(self) -> DBUtils:
+        return DBUtils()
+    
+    def test_sync_databases(self, database):
+        assert database.sync_databases()
+    
+    def test_fetch_headers(self, database):
+        assert database.get_headers()
+    
+    def test_fetch_data_gs(self, database):
+        assert database.get_all_data_gs()
+    
+    def test_fetch_data_sql(self, database):
+        assert database.get_all_data_sql()
+    
+    def test_fetch_users_gs(self, database):
+        assert database.get_all_users_gs()
+    
+    def test_fetch_users_sql(self, database):
+        assert database.get_all_users_sql()
+    
+    @pytest.mark.parametrize(
+            'update_type',
+            [
+                DatabaseUpdateType.ADD,
+                DatabaseUpdateType.EDIT,
+                DatabaseUpdateType.REMOVE
+            ]
+    )
+    def test_update_database(self, database, update_type: DatabaseUpdateType):
+        assert database.update_items_database(
+                update_type, [Item(
+                        *['None'] * 3,
+                        *[0] * 6,
+                        stock_manager.StockStatus.OUT_OF_STOCK.value
```

Comment:
In the dataclass here you should just hint it as the `StockStatus` enum, and pass the enum itself around.  rather than its value.  Doing it this way sort of defeats the purpose of setting up the enum